helpers/windowHistory.ts

```diff
@@ -1,19 +1,34 @@
 import CategoryEnum from '@/enums/Category.enum';
 
-export default function replaceQueryParams(_this: any, query: object): void {
-  if (!process.server) {
-    _this.$nuxt.$router.push({
-      path: _this.$nuxt.$route.path,
-      query,
+export default function replaceQueryParams(
+  _this: Record<string, any>,
+  query: object,
+): void {
+  _this.$router.push({
+    path: _this.$route.path,
+    query,
+  });
+}
+
+export function sameRouteReload(
+  _this: Record<string, any>,
+  route: string,
+  queryHook: string = 'in',
+): void {
+  if (_this.$route.path !== route) {
```

Comment:
nitpick(non-blocking): is this really the only way to do this? The 'in' will be displayed in the url shortly and then we have an immediate redirect.

Reply:
I'll do some more research today. Otherwise if I can't find another way to do this, the alternatives are:

- keep this workaround until we upgraded to vue-router v4
- ditch same page reload via navbar altogether as is the standard behavior in the current router version